pkg/jx/cmd/create_docker_auth.go

```diff
@@ -0,0 +1,138 @@
+package cmd
+
+import (
+	b64 "encoding/base64"
+	"encoding/json"
+	"github.com/jenkins-x/jx/pkg/jx/cmd/templates"
+	"github.com/jenkins-x/jx/pkg/util"
+	"github.com/spf13/cobra"
+	"gopkg.in/AlecAivazis/survey.v1"
+	"io"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	username = "user"
+	host     = "host"
+)
+
+type Config struct {
+	Auths map[string]*Auth `json:"auths,omitempty"`
+}
+
+type Auth struct {
+	Auth  string `json:"auth,omitempty"`
+	Email string `json:"email,omitempty"`
+}
+
+var (
+	createDockerAuthLong = templates.LongDesc(`
+		Creates/updates an entry for secret in the config.json for a given user, host
+`)
+
+	createDockerAuthExample = templates.Examples(`
+		# Create/update docker auth entry in the config.json file
+		jx create auth --host "angoothachap.private.docker.registry" --user "angoothachap" --secret "AngoothachapDockerHubToken" --email "fakeemail@gmail.com"
```

Comment:
Please use fake value such user `example` and so on. The current values seems to be PII.

Reply:
done.